themes/default/data/registry/packages/digitalocean.yaml

```diff
@@ -10,5 +10,5 @@ publisher: Pulumi
 repo_url: https://github.com/pulumi/pulumi-digitalocean
 schema_file_path: provider/cmd/pulumi-resource-digitalocean/schema.json
 title: DigitalOcean
-updated_on: 1675291056
-version: v4.17.0
+updated_on: -62135596800
```

Comment:
@susanev do you know how we (can) end up with a negative updated timestamp?

Reply:
i can't offer insight here, but @sean1588 looked into this same problem for another package last week.

Reply:
i'm not sure what is causing the negative timestamps tbh. i'll open an issue to look into it. I also wonder if this can just be set programatically by just looking up the timestamp of that specific release tag for that github repo.